### PR TITLE
Oracle driver param

### PR DESCRIPTION
**Pull request by cristianoteles**
I had to make this change in the code to use the library with Oracle.

### CODE REVIEW

**Comment by cossou** [comment]
Hi @cristianoteles 

thanks for the PR. 
Can you explain a bit what does `listParameters` and how it runs?

**Comment by cristianoteles** [comment]
Hi @cossou,
The ListParameters method returns the parameters that exist in the report. I will do a screen that the user choose which report will generate and I will list the parameters it needs to fill, reading this parameter the report itself.
I'm working on an example of use.

File: src/JasperPHP/JasperPHP.php [hunk 1]
```diff
@@ -127,11 +134,48 @@ public function execute($run_as_user = false)
...
+        $this->background       = false;
+        $this->the_command      = $command;
+
+        $parameters = array();
+        $response = $this->execute($run_as_user);
+
+        if ($response) {
+            foreach ($response as $param) {
+                $keywords = preg_split("/[\s,]+/", $param);
+                //P P_IDN_VEICULO java.lang.String
```
Comment (cossou) [inline src/JasperPHP/JasperPHP.php]: Can you remove or change this line? Thanks

**Comment by cossou** [comment]
@cristianoteles great function. Can you also add a small paragraph to the `Readme.md`.
Also, I merged an old PR and looks like there is a conflict with this one (I can try to solve this, don't worry).

**Comment by cristianoteles** [comment]
@cossou I removed the comment and put the example of use.

**Comment by cossou** [comment]
Hi @cristianoteles it's merged in the 2.0.0 version.

**Comment by cristianoteles** [comment]
Thanks

**Pull request closed without merging**